test/tutorial/step-3-setting-up-supertest.md

```diff
@@ -0,0 +1,19 @@
+### Setting up Supertest
```

Comment:
I'd reiterate what supertest is, in case people have forgotten

Reply:
This has also been added :)